submissions/Esebame1/Week 1

```diff
@@ -0,0 +1,11 @@
+for a in range (1,100):
+  if a%3==0 and a%5==0:
+    print('Fizzbuzz\n')
+     continue
+  elif a%3==0:
+    print('Fizz\n')
```

Comment:
Kindly delete this file Week 1 and update the PR. Changes here are duplicate of what is in `Week_1_solution.py`.

Reply:
Alright, I will. Thanks.